src/services/tokenPriceService.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable no-undef */
+const TOKEN_LIST_URL = "https://raw.githubusercontent.com/koindx/token-list/refs/heads/main/src/tokens/mainnet.json";
+const PRICE_API_BASE_URL = "https://koinoscollective.org/api/koindx/pair";
+const CHAINGE_USDT_ADDRESS = "14MjxccMUZrtBPXnNkuAC5MLtPev2Zsk3N";
```

Comment:
could you move this constant to `src/ts/storage.ts` ? this is the central place where we have all the addresses